testchain/runtime/src/lib.rs

```diff
@@ -907,9 +907,9 @@ impl_runtime_apis! {
 			VaultRegistry::get_collateralization_from_vault_and_collateral(vault, &amount, only_issued)
 		}
 
-		fn get_required_collateral_for_wrapped(amount_wrapped: BalanceWrapper<Balance>, currency_id: CurrencyId) -> Result<BalanceWrapper<Balance>, DispatchError> {
+		fn get_required_collateral_for_wrapped(amount_wrapped: BalanceWrapper<Balance>, _wrapped_currency_id: CurrencyId,  collateral_currency_id: CurrencyId) -> Result<BalanceWrapper<Balance>, DispatchError> {
 			let amount_wrapped = Amount::new(amount_wrapped.amount, GetWrappedCurrencyId::get());
```

Comment:
This was supposed to use the passed `wrapped_currency_id` instead of `GetWrappedCurrencyId::get()`. The `GetWrappedCurrencyId` was supposed to get removed as a whole.

Reply:
While `GetWrappedCurrencyId` is used in the chain spec for the default pairs, I think it would be better to move it there then as it is the only place where we really need it (assuming that you add the change to this function to use the passed `wrapped_currency_id`).

Reply:
So please move the definition of the default wrapped currency ID to the `chain_spec.rs` and remove it from here.

Reply:
Done.